sdks/java/core/src/main/java/org/apache/beam/sdk/util/BigQueryTableInserter.java

```diff
@@ -95,6 +94,22 @@ public BigQueryTableInserter(Bigquery client) {
     this.client = client;
```

Comment:
I think we just want to change the existing constructors to take in the additional pipelineoptions parameter.